app/views/publish/courses/_basic_details_tab.html.erb

```diff
@@ -75,7 +75,7 @@
      summary_list.with_row(html_attributes: { data: { qa: "course__funding" } }) do |row|
        row.with_key { "Funding type" }
        row.with_value { course.funding }
-       if course.is_published? || course.is_withdrawn?
+       if course.is_published? || course.is_withdrawn? || course.teacher_degree_apprenticeship?
```

Comment:
Will this need to change a year from now when TDA courses could be `draft_or_rolled_over? How will we remember to do that? I might be misunderstanding -- if so, it might be worth making a comment in the code as to why a tda course is being treated the same as a published or withdrawn course. The first (published and withdrawn) are states, where tda is type of course, so it's a bit confusing.

Reply:
Done

Reply:
Thanks @elceebee that's a great suggestion!